src/components/Inventory/withdraw/flow/SelectPurposeForm.tsx

```diff
@@ -325,6 +353,14 @@ export default function SelectPurposeForm(props: SelectPurposeFormProps): JSX.El
                   <FormControlLabel value={DEAD} control={<Radio />} label={strings.DEAD} />
                   <FormControlLabel value={OTHER} control={<Radio />} label={strings.OTHER} />
                 </RadioGroup>
+                {noReadySeedlings && (
+                  <Box display='flex' flexDirection='row' alignItems='center'>
+                    <Icon name='error' className={classes.errorIcon} />
+                    <Typography marginLeft={1} fontWeight={500} fontSize='14px' color={theme.palette.TwClrTxtDanger}>
+                      {strings.OUTPLANTS_REQUIRE_READY_SEEDLINGS}
+                    </Typography>
+                  </Box>
```

Comment:
@constanzauanini should we make this a new component? free standing error message that isn't tied to another component like in the other cases.

Reply:
Yes, we can make a new component. We may need it later

Reply:
Thanks, updated PR. I can move this into web-components as a follow-up after we merge visual updates.